e2e/wdio/driver/wdio.ts

```diff
@@ -61,7 +61,21 @@ export function goBack(): void {
     browser.back();
 }
 
-export function refreshPage(): void {
+export function refreshPage(isFullRefresh = false): void {
+    try {
+        browser.dismissAlert();
+    } catch {}
+    if (!isFullRefresh) {
+        const url = browser.getUrl();
+        click(`#toolbar-home-btn`);
+        if (browser.getUrl().includes(`/home`)) {
+            goBack();
+            return;
```

Comment:
If we're in this branch we're omitting `if (browserIsSafari())`, is it intended?

Reply:
I'm wondering why they actually added that extra pause for safari, probably it was a bug somewhere, it works fine for me even without it. But it definitely not needed, when we're basically clicking urls on the page, so yes, it's intended.